test/Microsoft.Dnx.Testing.CommonUtils/Solution.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Dnx.Runtime;
+using ProjectResolver = Microsoft.Dnx.Tooling.ProjectResolver;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class Solution
+    {
+        public Solution(string rootPath)
+        {
+            RootPath = rootPath;
```

Comment:
I certainly can. In fact I think the check would also be good to have in Dir. Note that usually a test should not need to create a Solution object itself and should use GetSolution() to grab a pre-constructed solution instead. Though I suppose a test could theoretically construct a solution using Dir (like in DnxPublishTests) and then create a Solution object to represent it, it is sort of redundant in that scenario.